src/tests/TestMain.lean

```diff
@@ -37,9 +37,16 @@ def testParser (dir : System.FilePath) (fn : Lean.Parser.ParserFn) : Config →
     updateExpected := config.updateExpected
   }
 
+def testManualTeX (dir : System.FilePath) : Config → IO Unit := fun config =>
+  Verso.GoldenTest.runTests {
+    testDir := ("src/tests/manual-tex" : System.FilePath) / dir,
+    runTest := fun _s => Verso.ManualTexTest.run -- TODO(admit multiple files)
```

Comment:
It seems to me that we should have a different test runner for this. The one used for the parser is really intended for friendlier testing of pure functions, while here I think we really want to check the whole run. What if this test were instead one that ran a command in a directory and then compared a given list of files to their expected versions? Then we could use the build command for the document unmodified, making it a better integration test, and the test running code could stay simpler and more to the point.